### ISSUE TITLE
[Bug]: Auth failed with error: 401

### ISSUE THREAD

**Issue by jaxonashton** [Brandawg93/homebridge-fordpass#186]
### Describe the bug

Getting error 401 again with the plugin after updating to the latest version. This issue was previously reported by other users under #162 and #165.

### Debug Output

```
[21/06/2022, 12:02:57] [homebridge-fordpass] Launched child bridge with PID 10715
[21/06/2022, 12:02:57] [homebridge-fordpass] Loaded homebridge-fordpass v1.6.1 child bridge successfully
[21/06/2022, 12:00:28] [homebridge-fordpass] Auth failed with error: 401
```

### Steps to reproduce

Not sure exactly how to reproduce. My login credentials are working on the website and the native Fordpass app, just not with the plugin. I attempted to remove and re-add my credentials, but no joy so far.

### Device Type

Raspi 4B

### iOS Version

15.5

### Bug Persistence

Consistently

### Last Working Version

1.5.3

**Comment by Brandawg93** [comment]
So it seems to be working for you now?

**Comment by Falc0n2k** [comment]
@Brandawg93 -- Correct. I am able to lock/unlock and remote start from the HomeKit with no issues.

I think what happened was a bad rollback I did last night that somehow borked the authentication path, even though I rolled back to v1.6.1. I had to wind up re-installing some other plugins as well.

It's been a huge PITA getting this IPv6 tunnel to cooperate with my whole network. I'll close this out.
